stacks/portstack.go

```diff
@@ -427,8 +440,9 @@ func (ps *PortStack) handleEth(dst []byte) (n int, err error) {
 			if err != nil {
 				return 0, err
 			} else if n > 0 {
+		
 				if isDebug {
-					ps.debug("UDP:send", slog.Int("plen", n))
+					ps.debug("UDP:send", slog.Int("plen", n))					
```

Comment:
This definetly should have been caught by gofmt, please make sure your editor runs gofmt on Go file save. VSCode does this by default.

Reply:
or not .. despite being set for both the user and the workspace, go selected as the formatter , and restarting vscode

<img width="1109" alt="image" src="https://github.com/user-attachments/assets/cd30154b-8651-439d-9992-a897bd470dec">

Manual formatting with alt,shift F works

Reply:
Hmm, I don't have that enabled. It seems Go formatting is independent of that option. See my JSON comment.

Reply:
got you